docs/experimental-features.md

```diff
@@ -365,3 +365,38 @@ ipfs config --json Experimental.ShardingEnabled true
 
 - [ ] Make sure that objects that don't have to be sharded aren't
 - [ ] Generalize sharding and define a new layer between IPLD and IPFS
+
+---
+
+## IPNS pubsub
+
+### In Version
+
+0.4.14
+
+### State
+
+Experimental, default-disabled.
+
+Utilizes pubsub for publishing ipns records in real time.
+
+When it is enabled:
+- IPNS publishers push records to a name-specific pubsub topic,
+  in addition to publishing to the DHT.
+- IPNS resolvers subscribe to the name-specific topic and receive newly
```

Comment:
At the moment, it's a bit unclear what this actually means, I'd go with something like:

> IPNS resolvers subscribe to name-specific topics on first resolutions to speedup future resolutions of the same name.

Reply:
i'll massage it a bit.